### ISSUE TITLE
Need to add repo before installing BerkeleyDB STL

### ISSUE THREAD

**Issue by fireballpoint1** [tdhock/PeakSegPipeline#3]
while running the following command 
`sudo aptitude install build-essential libdb6.0++-dev libdb6.0-stl-dev `
on a ubuntu16.04 system( I do have root access ), I faced the following error : 
```
build-essential is already installed at the requested version (12.1ubuntu2)
Couldn't find any package whose name or description matched "libdb6.0++-dev"
Couldn't find any package whose name or description matched "libdb6.0-stl-dev"
build-essential is already installed at the requested version (12.1ubuntu2)
Couldn't find any package whose name or description matched "libdb6.0++-dev"
Couldn't find any package whose name or description matched "libdb6.0-stl-dev"
```

I am guessing this is because we haven't added the appropriate repo before installing. Please provide the same or correct me in any other way

**Comment by fireballpoint1** [comment]
I was however able to install both the libraries for a depreciated version i.e. 5.3
` sudo aptitude install build-essential libdb5.3++-dev libdb5.3-stl-dev`

Can someone explain why?

**Comment by tdhock** [comment]
I think that BDB STL 5.3 is the only version available on ubuntu 16.04.

PeakSegPipeline should work with BDB STL 5.3 I think, so maybe you could update the PeakSegPipeline FAQ https://github.com/tdhock/PeakSegPipeline/wiki/FAQ to clarify any confusion that you had?

Also you may want to file a bug with the ubuntu devs (ask them to include the newer version of BDB STL dev)
